app/assets/v2/js/shared.js

```diff
@@ -1498,6 +1498,16 @@ $(document).ready(function() {
   });
 });
 
+const copyToClipboard = str => {
```

Comment:
don't know if exactly what you need but there is https://github.com/gitcoinco/web/blob/master/app/assets/v2/js/clipboard.js

Reply:
hmmm yeah this function is somehwat different (creates a diff message in a diff way, my `copyToClipBoard()` file, which is more generlizaable...